client/pom.xml

```diff
@@ -129,5 +121,11 @@
           <version>3.0.1</version>
           <scope>provided</scope>
       </dependency>
+      <dependency>
+          <groupId>junit</groupId>
+          <artifactId>junit</artifactId>
+          <version>4.12</version>
```

Comment:
IIRC there is version in dependency management, so you can remove the explicit definition

Reply:
> so you can remove the explicit definition

I don't believe I can. I tried doing it, and I got these errors:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.2:testCompile (default-testCompile) on project swarm-client: Compilation failure: Compilation failure: 
[ERROR] /Users/basil/work/third-party/jenkinsci/swarm-plugin/client/src/test/java/hudson/plugins/swarm/ClientTest.java:[4,17] package org.junit does not exist
```

Reply:
hm... will check locally. Not a big deal anyway